### ISSUE TITLE
Typo when adding Medip enrichment values

### ISSUE THREAD

**Issue by daonslog** [cruk-mi/mesa#27]
When running `mesa::addMedipsEnrichmentFactors`, the `typeString` does not correlate with the `nonEnrich` flag. 

```
function (qseaSet, exportPath = NULL, nonEnrich = FALSE, extend = 0, 
  shift = 0, uniq = 0, chr.select = NULL, paired = TRUE, file_name = "file_name", 
  nCores = 1) 
{
  BSgenome = qseaSet %>% qsea:::getGenome()
  if (!requireNamespace("MEDIPS", quietly = TRUE)) {
    stop("Package \"MEDIPS\" must be installed to use this function.", 
      call. = FALSE)
  }
  if (nonEnrich) {
    typeString <- "Pulldown" # should be "Input" if truly using the nonEnriched bams
  }
  else {
    typeString <- "Input" # should be "Pulldown" if truly NOT using the nonEnriched bams
  }
  ...
  return(qseaSet)
}
```

This leads to confusing logs / output messages e.g.:

```
> qseaSetCombined <- mesa::addMedipsEnrichmentFactors(qseaSetCombined,
+                                                     nCores = ifelse(parallel, BiocParallel::bpworkers(), 1),
+                                                     nonEnrich = FALSE)
Adding Medips Enrichment factors to 4 Input samples, using 38 cores. # should read "4 Pulldown samples"
```

**Comment by SPPearce** [comment]
Yes, this is a bug, have made a PR to fix it.
Thanks for reporting!

**Comment by SPPearce** [comment]
Fixed in #28
